spec/models/protected_food_drink_name_spec.rb

```diff
@@ -1,27 +0,0 @@
-require "spec_helper"
-require "models/valid_against_schema"
-
-RSpec.describe ProtectedFoodDrinkName do
-  subject(:payload) { FactoryBot.create(:protected_food_drink_name) }
-  include_examples "it saves payloads that are valid against the 'specialist_document' schema"
-
-  it "is not exportable" do
-    expect(described_class).not_to be_exportable
-  end
-
-  context "with Greek characters" do
-    subject(:document) { ProtectedFoodDrinkName.new(title: "Στερεά Ελλάδα") }
-
-    it "has a valid base_path" do
-      expect(document.base_path).to eq("/protected-food-drink-names/sterea-ellada")
-    end
-  end
-
-  context "with Cyrillic characters" do
-    subject(:document) { ProtectedFoodDrinkName.new(title: "Черноморски район") }
-
-    it "has a valid base_path" do
-      expect(document.base_path).to eq("/protected-food-drink-names/chiernomorski-raion")
```

Comment:
Hmmm we don't have an equivalent for that sort of test, do we? May need to keep.

And if it isn't specific to this finder (from a quick look, doesn't seem to be) then we should also consider moving this into a generic test somewhere.